clients/js/src/cipher.ts

```diff
@@ -80,6 +80,24 @@ export abstract class Cipher {
     return { data, nonce };
   }
 
+  /** Decrypts the data contained within call */
+  public async decryptCallData(
+    nonce: Uint8Array,
+    ciphertext: Uint8Array,
+  ): Promise<Uint8Array> {
+    const { body: plainData } = cbor.decode(
+      await this.decrypt(nonce, ciphertext),
+    );
+    return plainData;
+  }
+
+  /** Encrypts call result */
+  public async encryptCallResult(result: CallResult): Promise<Uint8Array> {
```

Comment:
This looks like it's just useful for testing and internal libraries, so I'm hesitant to expose this as the public interface of this library. Let's try something like
```suggestion
  /**
   * @hidden Encrypts a CallResult in the same way as would be returned by the runtime.
   * This method is not part of the SemVer interface and may be subject to change. 
   */
  public async encryptCallResult(result: CallResult): Promise<Uint8Array> {
```